mount/mountinfo_linux.go

```diff
@@ -81,11 +81,11 @@ func parseInfoFile(r io.Reader) ([]Info, error) {
 		p.Major, _ = strconv.Atoi(mm[0])
 		p.Minor, _ = strconv.Atoi(mm[1])
 
-		p.Root, err = strconv.Unquote(`"` + fields[3] + `"`)
+		p.Root, err = strconv.Unquote(`"` + strings.Replace(fields[3], `"`, `\"`, -1) + `"`)
 		if err != nil {
 			return nil, errors.Wrapf(err, "parsing '%s' failed: unable to unquote root field", fields[3])
 		}
-		p.Mountpoint, err = strconv.Unquote(`"` + fields[4] + `"`)
+		p.Mountpoint, err = strconv.Unquote(`"` + strings.Replace(fields[4], `"`, `\"`, -1) + `"`)
```

Comment:
Would switching to backquote solve this out the extra replace. https://golang.org/pkg/strconv/#Unquote
```
`"` -> "`"
```

Unless of course backquote also needs to be supported...

Reply:
> Would switching to backquote solve this out the extra replace. https://golang.org/pkg/strconv/#Unquote
> 
> ```
> `"` -> "`"
> ```
> 
> Unless of course backquote also needs to be supported...

It seems so:

```
› cat /proc/self/mountinfo | grep "foo\|bar"
1047 30 253:1 /tmp/`foo` /tmp/bar rw,relatime shared:1 - ext4 /dev/mapper/ubuntu--vg-root rw,errors=remount-ro
```

I added a commit with test case containing backticks as well.

Reply:
with backticks, `\040` is not expanded and this brings back the bug that the initial `unquote` solves.